### PR TITLE
10 delete comment

### PR DESCRIPTION
**Pull request by tuttitaken01**
task 10 done
edited task 8 based on feedback
task 9 had already done (comment count added to /api/reviews/:review_id endpoint)

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -494,3 +506,13 @@ describe("8.getReviews using queries", () => {
     })
 })
 
+describe("10.delComment", () => {
+    describe("DELETE /api/comments/commId", () => {
```
Comment (Hard-Luck) [inline __tests__/app.test.js:510]: need test for a ID that doesnt exist and an id of the wrong type

File: controllers/controllers.js [hunk 2]
```diff
@@ -166,4 +170,24 @@ exports.getUsername = (req, res, next) => {
     .catch(err => {
         next(err);
     })
+}
+
+exports.delComment = (req, res, next) => {
+    const id = req.params["commId"];
```
Comment (Hard-Luck) [inline controllers/controllers.js:177]: This should be extracted to the model, the controller should handle the request response and the model should handle all the data queries and formatting

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
Just need some more tests and a refactor

**Comment by two-triangles** [comment]
Hey Valentina, you haven't addressed Mark's comment regarding making your model dealing with interactions to the database. 

Please remove any console.logs that need to be removed.

**Pull request merged**